src/test/kotlin/com/terraformation/backend/db/DatabaseBackedTest.kt

```diff
@@ -2091,11 +2090,7 @@ abstract class DatabaseBackedTest {
     return rowWithDefaults.id!!
   }
 
-  fun insertTimeZone(timeZone: Any = ZoneId.of("Pacific/Honolulu")): ZoneId {
-    val zoneId = if (timeZone is ZoneId) timeZone else ZoneId.of("$timeZone")
-    timeZonesDao.insert(TimeZonesRow(zoneId))
-    return zoneId
-  }
+  fun insertTimeZone(timeZone: Any = ZoneId.of("Pacific/Honolulu")): ZoneId = ZoneId.of("$timeZone")
```

Comment:
Since time zones are already inserted, just return back the supplied or default value. I went this route so I didn't have to go through all test code and replace `insertTimeZone` calls with a hard coded value, if you'd rather I do that just let me know.